tests/system/python/conftest.py

```diff
@@ -668,6 +670,29 @@ def _disable_sch(fledge_url, sch_name):
     return _disable_sch
 
 
+@pytest.fixture(scope="function", autouse=True)
+def collect_support_bundle(request, fledge_url):
+    def _collect_support_bundle():
+        PROJECT_ROOT = Path(__file__).absolute().parent.parent.parent.parent
```

Comment:
This is wrong practice. We should avoid this and maybe use **Pathlib** which is a native Python library for handling files and paths and also provides a making code more readable and maintainable.

Reply:
We are currently utilizing the Pathlib library, given that the Path() function is associated with this library. As an alternative approach, we could use the "git rev-parse --show-toplevel" command to pinpoint the precise path of the PROJECT_ROOT. However, I will explore more optimal methods to determine this path.

Reply:
Try this code snippet to get PROJECT_ROOT 
```
import pathlib
PROJECT_NAME="tests"
PROJECT_ROOT = str(pathlib.Path().resolve()).split(PROJECT_NAME)[0]
print(PROJECT_ROOT)
```